src/Tizen.Wearable.CircularUI.Forms.Renderer/CircularUI.cs

```diff
@@ -44,5 +61,23 @@ public static void Init(string apiKey)
 
             Init();
         }
+
+        public static void Init(CoreApplication context)
+        {
+            var resPath = context?.DirectoryInfo?.Resource;
```

Comment:
```suggestion
            var resPath = Applications.Application.Current.DirectoryInfo.Resource;
```

Reply:
If you want to use `Applications.Application.Current` instead, we don't need to add additional `Init()` and `InitOptions` as well.

Reply:
`Applications.Application.Current` it can only use after `Application.Run()`

Reply:
Any other reason we should use `Applications.Application.Current` instead of context?